designer/client/page-create.js

```diff
@@ -224,7 +224,7 @@ class PageCreate extends React.Component {
               onChange={this.onChangeLinkFrom}
             >
               <option />
-              {pages.map((page) => (
+              {pages?.map((page) => (
```

Comment:
if these are optional wont the map fail?

Reply:
I ran `yarn jest` and it does fail without the optional operator now that we are using [context](https://github.com/XGovFormBuilder/digital-form-builder/pull/747/files#diff-f35316795696b7881627bd85f64126226a34e2e80db706d7a79a5a434df8b13e), it shouldn't result in any unexpected behaviour

Reply:
if pages are undefined (or null) it will stop executing, so I think this one will be ok